src/traccuracy/loaders/_point.py

```diff
@@ -0,0 +1,90 @@
+from typing import Optional, cast
+
+import networkx as nx
+import pandas as pd
+
+from traccuracy._tracking_graph import TrackingGraph
+
+
+def load_point_data(
+    path: str | None = None,
+    df: Optional[pd.DataFrame] = None,  # noqa: UP007, bar syntax breaks docks build
+    parent_column: str = "parent",
+    id_column: str | None = None,
+    pos_columns: tuple[str, ...] = ("z", "y", "x"),
+    time_column: str = "t",
+    name: str | None = None,
+    sep: str | None = None,
+) -> TrackingGraph:
+    """Load point-based tracking data into a TrackingGraph from a csv-like file
+
+    Assumes each row contains:
+    - time
+    - position, e.g. three columns 'z', 'y', 'x'
+    - parent, a reference to the node in the previous time frame
+
+    Args:
+        path (str | None, optional): Path to the csv-like file to load. Defaults to None.
+        df (pd.DataFrame | None, optional): A dataframe that has already been loaded.
+            Defaults to None.
+        parent_column (str | None, optional): A reference to the parent node in the previous
+            time frame. Defaults to "parent".
+        id_column (str | None, optional): Optional column used to specify node ids.
```

Comment:
I don't think the default being None makes sense combined with "parent." Whoever wrote out the parent ID would have to know the default that you picked (row index) and there is a big risk of mess if that's not right. I think we should require a node id column.